shared/src/utils/parsePseudoCode.ts

```diff
@@ -0,0 +1,555 @@
+import {
+  PseudoCode,
+  PseudoCodeAssignment,
+  PseudoCodeBlock,
+  PseudoCodeCall,
+  PseudoCodeFor,
+  PseudoCodeForAll,
+  PseudoCodeFunction,
+  PseudoCodeFunctionName,
+  PseudoCodeIf,
+  PseudoCodePrint,
+  PseudoCodePrintString,
+  PseudoCodeReturn,
+  PseudoCodeState,
+  PseudoCodeString,
+  PseudoCodeVariable,
+  PseudoCodeWhile,
+} from "@shared/question-generators/time/pseudoCodeUtils.ts"
+import {
+  controlFlowColor,
+  functionColor,
+  keyWordsColor,
+  printStatementColor,
+  variableColor,
+} from "@/components/DrawPseudoCode.tsx"
```

Comment:
In the long run we should think about where to put the colors. There should be a folder with style definitions.